packages/terra-form-select/src/shared/_Option.module.scss

```diff
@@ -77,6 +77,7 @@
 
   .is-active {
     background-color: var(--terra-form-select-option-active-background-color, #ebf6fd);
+    border: var(--terra-form-select-option-active-color-border, 2px dashed #000);
```

Comment:
this should be outline and outline-offset I think not a border !!

Reply:
Updated